### PR TITLE
fix the gap between github button icon and dark mode button of top Navbar , Please check

### PR DESCRIPTION
**Pull request by avinashji1302**
<h1>fix the gap between top navbar  github icon button and dark mode button</h1>

<h1>Please check and if there is any problem ... Please let me know<h1/>

### CODE REVIEW

File: src/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -101,7 +102,7 @@ export default function NavBar() {
                     </button>
 
                     {/* Profile dropdown */}
-                    <Menu as="div" className="relative ml-3">
+                    <Menu as="div" className="relative ml-3 p-4">
```
Comment (mathcodes) [inline src/components/navbar/Navbar.jsx:105]: Suggestion: Lets take out ml-3, that way the three icons sit in an evenly distributed row. Thoughts?

Comment (mathcodes) [inline src/components/navbar/Navbar.jsx:105]: See suggestion. I'm going to accept and merge this PR, but if you want another green square, remove that ml-4. Also, make sure to add your info to the contributor's section in the README and the App itself.

**Review by mathcodes — APPROVED** [review]
Good touch. Submit another PR after adding your github info the contributor's sections in the README as well as the APP itself, and remove the ml-4 as discussed in the comments. Let me know if you have any questions and thanks for being part of the team!!!

**Comment by avinashji1302** [comment]
> Good touch. Submit another PR after adding your github info the contributor's sections in the README as well as the APP itself, and remove the ml-4 as discussed in the comments. Let me know if you have any questions and thanks for being part of the team!!!

Thank You for merging it and I am going to remove ml-4 and update myself in Readme file

**Pull request merged**